packages/generator/src/vdm-types.ts

```diff
@@ -83,11 +88,12 @@ export interface VdmComplexType {
 export interface VdmFunctionImport {
   httpMethod: string;
   originalName: string;
-  returnType: VdmFunctionImportReturnType;
   parameters: VdmParameter[];
   parametersTypeName: string;
   functionName: string;
   description: string;
+  returnTypeEdmx: string;
```

Comment:
With this we have duplicate information - VdmFunctionImportReturnTypeCategory.

Reply:
also a symptom of the bad return type handling. Now removed after the factory name for the comple type is left.